docs/BUILD.bazel

```diff
@@ -364,6 +365,10 @@ genrule(
         mkdir -p build/docs/theme
         tar -zxf $(location :theme) -C build/docs/theme
 
+        # Copy templates for code snippets in getting started guide
+        mkdir -p build/docs/source/getting-started/code
+        tar -zxf $(location //templates:templates-tarball) -C build/docs/source/getting-started/code/
```

Comment:
It would be better to only put the sources of the `create-daml-app` template here. I'm fine with improving this in a followup PR.